packages/host/src/Host.tsx

```diff
@@ -223,7 +223,7 @@ const Host: FC = () => (
                           overflow="hidden"
                         >
                           <AppBar />
-                          <Box display="flex" flex={1} overflow="scroll">
+                          <Box display="flex" flex={1} overflow="auto">
```

Comment:
this is trying to make the scroll bar on the RHS go away when there's no scrolling to do